Homework - 02 Basic Python.ipynb

```diff
@@ -29,11 +64,81 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 14,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[3]\n",
+      "do you want another card, yes or no: urd\n",
+      "do you want another card, yes or no: yes\n",
+      "[3, 6]\n",
+      "do you want another card, yes or no: no\n",
+      "computer hand: [6, 7]\n",
+      "user hand: [3, 6]\n",
+      "game over. computer wins!\n"
+     ]
+    }
+   ],
    "source": [
-    "# your code"
+    "import random\n",
+    "\n",
+    "computer = []\n",
+    "user = []\n",
+    "\n",
+    "computer.append(random.randrange(1,11))\n",
+    "user.append(random.randrange(1,11))\n",
+    "print(user)\n",
+    "\n",
+    "while True:\n",
+    "    game_over = input('do you want another card, yes or no: ')\n",
+    "    if game_over == \"no\":\n",
+    "        if sum(user) > sum(computer):\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print (\"game over. user wins!\")\n",
+    "            break\n",
+    "        if sum(user) < sum(computer):\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"game over. computer wins!\")\n",
+    "            break\n",
+    "    \n",
+    "    if game_over == \"yes\":\n",
+    "        computer.append(random.randrange(1,11))\n",
+    "        user.append(random.randrange(1,11))\n",
+    "        print(user)\n",
+    "        \n",
+    "        if sum(computer) == 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"21! computer wins!\")\n",
+    "            break\n",
+    "        elif sum(user) == 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"21! you win!\")\n",
+    "            break\n",
+    "        elif sum(user) > 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"you lose ):\")\n",
+    "            break\n",
+    "        elif sum(computer) > 21:\n",
+    "            print(\"computer hand: \" + str(computer))\n",
+    "            print(\"user hand: \" + str(user))\n",
+    "            print(\"computer over 21. you win by default!\")\n",
+    "            break\n",
+    "        elif where_user < 21:\n",
```

Comment:
what is `where_user`? that doesn't seem to be defined anywhere? Python should throw an error here.